iceberg/iceberg-handler/src/test/results/positive/col_stats.q.out

```diff
@@ -339,17 +339,16 @@ POSTHOOK: type: DESCTABLE
 POSTHOOK: Input: default@tbl_ice_puffin
 col_name            	a                   
 data_type           	int                 
-min                 	1                   
-max                 	333                 
-num_nulls           	0                   
-distinct_count      	7                   
+min                 	                    
+max                 	                    
+num_nulls           	                    
+distinct_count      	                    
 avg_col_len         	                    
 max_col_len         	                    
 num_trues           	                    
 num_falses          	                    
-bit_vector          	HL                  
+bit_vector          	                    
 comment             	                    
-COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\",\"COLUMN_STATS\":{\"a\":\"true\",\"b\":\"true\",\"c\":\"true\"}}
```

Comment:
Basic stats are removed as well?

Reply:
Same as `min`, `max` stats above, the HMS thinks it's unable to provide the accurate stats due to the data files left behind before the table `tbl_ice_puffin` creation.